Xamarin.Forms.Core/ResourceDictionary.cs

```diff
@@ -5,19 +5,22 @@
 using System.Linq;
 using System.Reflection;
 using Xamarin.Forms.Internals;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
 
 namespace Xamarin.Forms
 {
 	public class ResourceDictionary : IResourceDictionary, IDictionary<string, object>
 	{
 		static ConditionalWeakTable<Type, ResourceDictionary> s_instances = new ConditionalWeakTable<Type, ResourceDictionary>();
 		readonly Dictionary<string, object> _innerDictionary = new Dictionary<string, object>();
-
+		ResourceDictionary _mergedInstance;
 		Type _mergedWith;
+
 		[TypeConverter (typeof(TypeTypeConverter))]
 		public Type MergedWith {
```

Comment:
Maybe we should deprecate this?

Reply:
we will, but we need to implement `Source` for that